asyncio/queue.py

```diff
@@ -0,0 +1,145 @@
+# SPDX-FileCopyrightText: 2019-2020 Damien P. George
```

Comment:
# SPDX-FileCopyrightText: Copyright (c) 2023 Your Name Here / Handle? for Adafruit Industries ? What's the right approach here?

Reply:
Per @dhalbert I copied the original copyright

Reply:
"for Adafruit Industries" is when someone got paid to do the work.